sheepdog/transactions/upload/entity.py

```diff
@@ -163,6 +165,29 @@ def _get_node_create(self, skip_node_lookup=False):
                     type=EntityErrors.NOT_UNIQUE,
                 )
 
+        # Check to see if it's registered in dbGaP is is an exception
```

Comment:
verifying against dbgap should be optional
gen3 submission don't need to check with dbgap